pkg/apis/proto/daemon/daemon.proto

```diff
@@ -109,6 +109,24 @@ message GetPipelineWatermarksRequest {
   string pipeline = 1;
 }
 
+message GetVertexErrorsRequest {
+  string pipeline = 1;
+  string vertex = 2;
+  string replica = 3;
+}
+
+message VertexError {
+  string container = 1;
+  string timestamp = 2;
```

Comment:
Why use `string`?

Reply:
when we store the timestamp in json file in emptyDir we already convert it to string. Thats why when we pass it back to the numaflow server, we thought of passing it as a string.